roles/heat/templates/etc/heat/heat.conf

```diff
@@ -37,7 +37,11 @@ instance_connection_is_secure=1
 region_name_for_services=RegionOne
 
 rpc_thread_pool_size=64
+{% if ceilometer.enabled|default('True')|bool -%}
```

Comment:
I noticed in haproxy_openstack.cfg we have it set like this, was following suit for heat as well. I will change the default here to False for heat.